Rakefile.rb

```diff
@@ -1,30 +1,49 @@
 require 'json'
 require 'ohai'
 
-PROJECT_DIR='/dd-agent-omnibus'
-
 @ohai = Ohai::System.new.tap { |o| o.all_plugins(%w{platform}) }.data
 
+def linux?()
+  %w(rhel debian fedora suse gentoo slackware arch exherbo).include? @ohai['platform_family']
+end
+
+def osx?()
+  @ohai['platform_family'] == 'mac_os_x'
+end
+
+def windows?()
+  @ohai['platform_family'] == 'windows'
+end
+
+
+if windows?
+  PROJECT_DIR='c:\dd-agent-omnibus'
+  FSROOT="/c/"
```

Comment:
It is cygwin syntax, which is why it needs to be used in some places.  The rm command, at least, seems to be a cygwin binary and requires it.